src/instructlab/configuration.py

```diff
@@ -501,11 +530,30 @@ def ensure_storage_directories_exist():
         DEFAULTS.INTERNAL_DIR,
         DEFAULTS.MODELS_DIR,
         DEFAULTS.TAXONOMY_DIR,
+        DEFAULTS.TRAIN_CONFIG_DIR,
+        DEFAULTS.TRAIN_PROFILE_DIR,
+        DEFAULTS.TRAIN_EXPERT_OPTIONS_DIR,
     ]
 
     for dirpath in dirs_to_make:
         os.makedirs(dirpath, exist_ok=True)
 
+    expert_args_and_defaults = {
+        "learning_rate": 2e-6,
+        "warmup_steps": 1000,
+        "random_seed": 42,
+        "node_rank": 0,
+        "nnodes": 1,
+        "nproc_per_node": 1,
+        "rdzv_id": 123,
+        "rdzv_endpoint": "127.0.0.1:12222",
+        "is_padding_free": False,
```

Comment:
Notes: 
- is_padding_free should get moved out
- learning_rate == 2e-5
- warmup_steps == 25
- nproc_per_node should move out of expert args